edn/build.rs

```diff
@@ -0,0 +1,6 @@
+
+extern crate peg;
+
+fn main() {
+    peg::cargo_build("src/edn.rustpeg");
```

Comment:
Hm. I suppose I was thinking that Rust syntax highlighting, indenting, etc. would be better than nothing? Up to you.

There are some syntax highlighting plugins for it:

https://github.com/treycordova/rustpeg.vim

with filetype `set filetype=rust.rustpeg`, so perhaps just add that as a Vim modeline and hope Linguist figures it out?